### PR TITLE
2021 markup japanese translation

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of #586

I update markup of english.

### CODE REVIEW

File: src/content/en/2021/markup.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ In terms of popularity, HTML5, better known as `<!DOCTYPE html>` is still the mo
   ) }}</figcaption>
 </figure>
 
-The surprising part is that, <a hreflang="en" href="https://en.wikipedia.org/wiki/XHTML">almost 20 years later</a>, XHTML is still a considerable part of the web, with 8% of pages still using it on desktop and a little under 7% on mobile.
+The surprising part is that, [almost 20 years later](https://en.wikipedia.org/wiki/XHTML), XHTML is still a considerable part of the web, with 8% of pages still using it on desktop and a little under 7% on mobile.
 
 ### Document size
```
Comment (tunetheweb) [inline src/content/en/2021/markup.md:87]: Should this be translated?

File: src/content/ja/2021/markup.md [hunk 2]
```diff
@@ -0,0 +1,987 @@
...
+  caption="もっともポピュラーなボタンタイプです。",
+  description="使用頻度の高いボタンの種類トップ5を棒グラフで表示したもの。もっとも人気のあるものは、ボタンタイプ、タイプなし、送信タイプです。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSaFU4lh2yyjudn8fcy0l65EbP9MdCbB39wQbT0c5cfcJzfl2p1nGOByDdgg8P3dMeZMa0eWkkG685n/pubchart?oid=351242151&format=interactive",
+  sheets_gid="1847095902",
+  sql_file="buttons.sql"
+) }}
+
+[昨年](../2020/markup#button-and-input-types)と比較したところ、`button`要素のあるページがかなり増えていることがわかりました。今年は `input` 型のボタンに対するクエリは実行しませんでしたが、ページ上の `button` 要素の使用数は明らかに減少しています。[アクセシビリティの章](./accessibility)にもボタンに関するセクションがあるので、そちらも読んでみてください。
+
+### Links
```
Comment (tunetheweb) [inline src/content/ja/2021/markup.md:863]: Should this be translated?

**Review by tunetheweb — CHANGES_REQUESTED** [review]
Looks good. Two titles I think should be translated but let me know if you don't think they should be.

File: src/content/en/2021/markup.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/en/2021/markup.md:87]: Isn't this your commitment?
https://github.com/HTTPArchive/almanac.httparchive.org/pull/2741/commits/718b5b6dca0946eae798c2ee9feee00ada52c190

File: src/content/ja/2021/markup.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2021/markup.md]: Thanks, mistake.
I translated. 👍

File: src/content/en/2021/markup.md (hunk 1 above)
Comment (tunetheweb) [reply src/content/en/2021/markup.md:87]: I was talking about the “Document size” title for this comment. I fixed the nearby link in that other commit but still think the title is worthy of translating. What do you think?

Comment (ksakae1216) [reply src/content/en/2021/markup.md:87]: Sorry, I was wrong.
I translated. Thanks

**Pull request merged**